packages/demo/src/configuration/editor/blockEditorRegistrations.ts

```diff
@@ -181,4 +188,46 @@ export const blockEditorRegistrations: IBlockEditorRegistration[] = [
         category: "Effects",
         tooltip: "Alters the exposure of the input texture",
     },
+    {
+        name: "StarryPlanesBlock",
+        factory: (smartFilter: SmartFilter) => new StarryPlanesBlock(smartFilter, "StarryPlanes"),
+        category: "Generators",
```

Comment:
It may make sense to move the generators into a new folder named generators alongside effects so the folders match the catagories

Reply:
Done!